zappa/zappa.py

```diff
@@ -752,8 +749,8 @@ def create_and_setup_methods(self, restapi, resource, lambda_arn, api_key_requir
             integration.Credentials = credentials
             integration.IntegrationHttpMethod = 'POST'
             integration.IntegrationResponses = []
-            integration.PassthroughBehavior = 'NEVER'
-            integration.RequestParameters = {}
+            # integration.PassthroughBehavior = 'NEVER'
```

Comment:
Why did you drop passthrough behavior? I think that was intentionally added there to make API Gateway return `415 Unsupported Media ` for unexpected mime types..